src/components/App.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+// import logo from './logo.svg';
```

Comment:
don't leave comment

Reply:
i left comments to render them on the browser. With those undefined components, the code wouldnt run.